src/app.scss

```diff
@@ -222,3 +222,15 @@
         padding-block-start: 0;
     }
 }
+
+// // FIXME: Promote the CSS bedlow to overrides, open PF issues // //
+
+// PatternFly always adds a margin after images inside of widgets with pf-m-end, which is incorrect when it's the last element
+.pf-v5-c-button__icon.pf-m-start:last-child {
+    margin-inline-end: 0;
+}
```

Comment:
PF-React issue. It always adds margin, either at the beginning or end, regardless of if there is text or not. (It's after in the default.)

Here's the issue illustrated on the PF Docs website, on the React widget.

![image](https://github.com/user-attachments/assets/974c4621-6c9d-4952-bd42-97426757c9a3)